hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -764,6 +794,22 @@ public Target updateControllerAttributes(final String controllerId, final Map<St
         return targetRepository.save(target);
     }
 
+    @Override
+    @Transactional
+    @Retryable(include = {
+            ConcurrencyFailureException.class }, maxAttempts = Constants.TX_RT_MAX, backoff = @Backoff(delay = Constants.TX_RT_DELAY))
+    public Target updateControllerName(final String controllerId, final String controllerName) {
+
+        final JpaTarget target = (JpaTarget) targetRepository.findByControllerId(controllerId)
+                .orElseThrow(() -> new EntityNotFoundException(Target.class, controllerId));
+
+        target.setName(controllerName);
+
+        assertTargetAttributesQuota(target);
```

Comment:
This is not needed here, I guess.

Reply:
True, method is changes and also has void return type now.

Reply:
why are you returning a void type now? I think it's actually a nice feature to return the updated target, as you did in the first place.

Reply:
Because a return type is not needed by now and if it's needed in future, it can be adapted again :)